2-extra/2-water-bottle.js

```diff
@@ -18,18 +18,29 @@ You have to implement the missing features according to the specification.
 let bottle = {
   volume: 0,
   fillUp: function () {
+    this.volume = 100;
     // calling this function should completely fill your bottle (volume = 100);
   },
   pour: function () {
```

Comment:
Small point here - what happens if volume is 99, and then we call pour()?
In the comments above it says: `Note: You cannot exceed the bottle capacity.`
There is no test to check this case, so it will be a good idea to write a test first to see the issue, and then fix the issue after 😄